core/src/main/java/nl/nn/adapterframework/filesystem/ExchangeFileSystem.java

```diff
@@ -151,27 +150,44 @@ public class ExchangeFileSystem extends MailFileSystemBase<EmailMessage,Attachme
 	private @Getter String tenantId = null;
 
 	/* SSL */
-	private @Getter @Setter String keystore;
-	private @Getter @Setter String keystoreAuthAlias;
-	private @Getter @Setter String keystorePassword;
-	private @Getter @Setter KeystoreType keystoreType=KeystoreType.PKCS12;
-	private @Getter @Setter String keystoreAlias;
-	private @Getter @Setter String keystoreAliasAuthAlias;
-	private @Getter @Setter String keystoreAliasPassword;
-	private @Getter @Setter String keyManagerAlgorithm=null;
-
-	private @Getter @Setter String truststore=null;
-	private @Getter @Setter String truststoreAuthAlias;
-	private @Getter @Setter String truststorePassword=null;
-	private @Getter @Setter KeystoreType truststoreType=KeystoreType.JKS;
-	private @Getter @Setter String trustManagerAlgorithm=null;
-	private @Getter @Setter boolean allowSelfSignedCertificates = false;
-	private @Getter @Setter boolean verifyHostname=true;
-	private @Getter @Setter boolean ignoreCertificateExpiredException=false;
-	private @Getter @Setter boolean enableConnectionTracing=false;
-
-	private @Getter CredentialFactory credentials=null;
-	private @Getter CredentialFactory proxyCredentials=null;
+	private @Getter
+	@Setter String keystore;
+	private @Getter
+	@Setter String keystoreAuthAlias;
+	private @Getter
+	@Setter String keystorePassword;
+	private @Getter
+	@Setter KeystoreType keystoreType = KeystoreType.PKCS12;
+	private @Getter
+	@Setter String keystoreAlias;
+	private @Getter
+	@Setter String keystoreAliasAuthAlias;
+	private @Getter
+	@Setter String keystoreAliasPassword;
+	private @Getter
+	@Setter String keyManagerAlgorithm = null;
+
+	private @Getter
+	@Setter String truststore = null;
+	private @Getter
+	@Setter String truststoreAuthAlias;
+	private @Getter
+	@Setter String truststorePassword = null;
+	private @Getter
+	@Setter KeystoreType truststoreType = KeystoreType.JKS;
+	private @Getter
+	@Setter String trustManagerAlgorithm = null;
+	private @Getter
+	@Setter boolean allowSelfSignedCertificates = false;
+	private @Getter
+	@Setter boolean verifyHostname = true;
+	private @Getter
+	@Setter boolean ignoreCertificateExpiredException = false;
+	private @Getter
+	@Setter boolean enableConnectionTracing = false;
```

Comment:
Deze andere formattering vind ik geen verbetering

Reply:
@gvanbrakel Ik ben het helemaal met je eens. Is er per ongeluk doorheen geslipt. Formatter settings aangepast!